### PR TITLE
Update About Us page

### LINKED ISSUE #92: [Feature Request] About page enhancements
Issue Description
-----------------
- [ ] Ask on slack, and add the advisors
- [ ] Sponsor logos (share the component with the home page?)
- [ ] Contact us at `info@latticesurgery.com`
- [ ] A "press" section with the two articles on SFU's CS and Phys websites and the one at TUM

### PR DESCRIPTION
**Pull request by alexnguyenn**
Closes #92 

## Changes 
* Add `ProfileBox` and `Article` components. 
* Add advisors section.
* Add press section (mentioned in)

### CODE REVIEW

**Comment by alexnguyenn** [comment]
Still undecided about bringing the sponsors in home page over.

**Comment by gwwatkin** [comment]
The styles and pictures look good. Need to fill it up with some text. We can ask for it in slack.

As for the sponsors, we probably don't need them since they are all mentioned in the text at the top

File: src/pages/AboutUs.tsx [hunk 1]
```diff
@@ -1,149 +1,215 @@
...
+                />
+
+                <Profile
+                    name="Keelan Watkins"
+                    sub="Keelando"
+                    avatarSrc="https://avatars.githubusercontent.com/u/47836064"
+                    description={
+                        "Metabolizer of data through methodical application of interconnected " +
+                        "web technologies. When I'm not programming, I'm surfing."
+                    }
```
Comment (alexnguyenn) [inline src/pages/AboutUs.tsx:103]: @Keelando can you update the description here?

**Review by gwwatkin — APPROVED** [review]

**Review by Keelando — COMMENTED** [review]
All good. Updated my section. I still feel like its a bit awkward having the About Tab on the far right, and having to skip over the two external links. I propose we keep the external link tabs to the right

**Review by Keelando — APPROVED** [review]

**Comment by gwwatkin** [comment]
Sorry hit approve accidentally, messaged each prof to hear if they like their card or would like a particular phrase

**Review by gwwatkin — APPROVED** [review]

**Pull request merged**